### PR TITLE
Fix install instructions for MXNET-R

### PR DESCRIPTION
**Pull request by ankkhedia**
## Description ##
The PR fixes instructions on index page for MXNET-R installation for Windows(GPU support)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [x] Changing name of bucket storing R binaries

## Comments ##
-  This PR fixes the name of bucket storing new R binaries. It also fixes some broken links.

### CODE REVIEW

**Comment by ankkhedia** [comment]
@aaronmarkham @hetong007 Could you please take a look?

File: docs/install/index.md [hunk 1]
```diff
@@ -1797,14 +1797,15 @@ install.packages("mxnet")
 
 <div class="gpu">
 
-You can [build MXNet-R from source](windows_setup.html#install-the-mxnet-package-for-r), or you can use a pre-built binary:
+You can [build MXNet-R from source](windows_setup.html#install-mxnet-package-for-r), or you can use a pre-built binary:
 
 ```r
 cran <- getOption("repos")
-cran["dmlc"] <- "https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/R/CRAN/GPU"
+cran["dmlc"] <- "https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/R/CRAN/GPU/cuX"
```
Comment (hetong007) [inline docs/install/index.md]: suggest to write 

```
cran["dmlc"] <- "https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/R/CRAN/GPU/cu80"
```

as a concrete example, and then mention replacing `cu80` with `cu90`, `cu91`, `cu92` also works.

Comment (ankkhedia) [reply docs/install/index.md]: @hetong007 fixed as suggested by you.

File: docs/install/index.md [hunk 2]
```diff
@@ -1797,14 +1797,15 @@ install.packages("mxnet")
...
-You can [build MXNet-R from source](windows_setup.html#install-the-mxnet-package-for-r), or you can use a pre-built binary:
+You can [build MXNet-R from source](windows_setup.html#install-mxnet-package-for-r), or you can use a pre-built binary:
 
 ```r
-cran <- getOption("repos")
-cran["dmlc"] <- "https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/R/CRAN/GPU"
-options(repos = cran)
-install.packages("mxnet")
+  cran <- getOption("repos")
+  cran["dmlc"] <- "https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/R/CRAN/GPU/cu80"
```
Comment (aaronmarkham) [inline docs/install/index.md]: I would think that you would want to default to the recommended package, which isn't the old cu80, right? Why not make that cu92?

File: docs/install/windows_setup.md [hunk 3]
```diff
@@ -218,11 +218,11 @@ For GPU package:
 
 ```r
   cran <- getOption("repos")
-  cran["dmlc"] <- "https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/R/CRAN/GPU/cuX"
+  cran["dmlc"] <- "https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/R/CRAN/GPU/cu80"
```
Comment (aaronmarkham) [inline docs/install/windows_setup.md]: Default to cu92?

File: docs/install/index.md (hunk 2 above)
Comment (hetong007) [reply docs/install/index.md]: Makes sense.

Comment (ankkhedia) [reply docs/install/index.md]: Done

File: docs/install/windows_setup.md (hunk 3 above)
Comment (ankkhedia) [reply docs/install/windows_setup.md]: done

**Pull request merged**